Week-1/HTML Assignment1/index.html

```diff
@@ -0,0 +1,153 @@
+<html>
+  <head>
+	<link href='https://fonts.googleapis.com/css?family=Lato' rel='stylesheet'>
+    <style>	
+		body {
+			font-family: 'Lato';
+		}
+      .bottomLabel {
+        color: white;
+      }
+      .bottomIcon {
+        padding-right: 100px;
+        height: 12px;
+        width: 12px;
+      }
+      .bottomIcon1 {
+        padding-left: 1000px;
+        padding-right: 100px;
+        height: 12px;
+        width: 12px;
+      }
+      .vertical
+      {
+        margin:0;
+        padding:0;
+        height: 100%;
+        width:2%;
+        background-color: #B8B8B8;
+      }
+      .boldh
+      {
+        color:white;
+        font-size: small;
+      }
+      .iconWithLabel{
+        padding-right: 20px;
+        height:12px;
+        width:12px;
+        color: white
+       }
```

Comment:
Also install prettier plugin in your visual studio for formatting the code

Reply:
install and applied prettier code formatter